### DISCUSSION TITLE
Not working properly after building the app？

### DISCUSSION THREAD

**Discussion by ZeronoFreya** [wailsapp/wails#1692] [Q&A] [answered]
```
System
------
OS:             Windows 10 Home
Version:        2009 (Build: 22000)
ID:             21H2
Go Version:     go1.17.6
---------
Your system is ready for Wails development!
Optional package(s) installation details:
  - upx : Available at https://upx.github.io/
  - nsis : Available at https://nsis.sourceforge.io/Download
```
I tried building with "wails build", the interface loads but doesn't work properly, I can't see the problem.
But when I use "wails build --debug" everything works fine.
I can't locate the error correctly.
What should I do?

`WebView2 Runtime Version '103.0.1264.77' installed. Minimum version required: 91.0.992.28.`
Does this problem need to be solved? This seems to be a problem with Google.

**Comment by ianmjones** [comment, marked as answer]
Does everything work as expected when using `wails dev`?

If so, after running that, does the result of another `wails build` now work as expected?

I'm wondering whether it's just that the frontend code hasn't been (re)generated yet: https://github.com/wailsapp/wails/discussions/1683#discussioncomment-3295444

If using `wails dev` does work, you can also try `wails generate module` followed by `wails build`: https://wails.io/docs/next/reference/cli#module

If none of that helps, what version of wails are you using?

**Reply by ZeronoFreya** [reply]
As far as the result is concerned, it is a network failure. I used a local proxy. No problems were found in "dev" mode. After packaging, the proxy was blocked.
